Apps/WebClient/src/ClientApp/src/components/report/ReportsComponent.vue

```diff
@@ -294,6 +368,13 @@ export default class ReportsComponent extends Vue {
             return;
         }
 
+        if (this.isDependent) {
+            SnowPlow.trackEvent({
+                action: "download_report",
+                text: `Dependent_${this.reportComponentName}`,
+            });
+        }
+
         this.isGeneratingReport = true;
 
         this.trackDownload();
```

Comment:
This should be integrated into `trackDownload()` so that only one event is issued when dowloading a dependent report.  You should also probably use the same report names it uses (but with "Dependent " or "Dependent_" in front).